android/titanium/src/java/org/appcelerator/titanium/view/TiActivitySafeAreaMonitor.java

```diff
@@ -105,6 +108,14 @@ public void onLayoutChange(View view, int left, int top, int right, int bottom,
 			@Override
 			public WindowInsets onApplyWindowInsets(View view, WindowInsets insets)
 			{
+				if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
```

Comment:
Move the listener-check to here

Reply:
moved tiapp into the safeareamonitor file and doing the check there now